.github/workflows/nodejs.condo.ci.yml

```diff
@@ -0,0 +1,54 @@
+name: Condo CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  build-image:
+    name: Build Docker Image
+    runs-on: runners-dind-set-cpu5-ram10
+    outputs:
+      DOCKER_IMAGE_FULL: ${{ env.DOCKER_IMAGE_FULL }}
+    steps:
+      - name: Prepare runner
+        run: |
+          sudo apt update && sudo apt install -y git
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          submodules: recursive
+          ssh-key: ${{ secrets.SSH_DOCK_SERVER_PRIVATE_KEY }}
+      - name: Setup context for Docker BuildX
+        id: buildx-context
+        run: |
+          docker context create builders
+      - name: Setup Docker BuildX
+        id: buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          endpoint: builders
+      - name: Login to cloud registry
+        uses: docker/login-action@v3
+        with:
+          registry: swr.ru-moscow-1.hc.sbercloud.ru
+          username: ${{ secrets.SBERCLOUD_CR_USERNAME }}
+          password: ${{ secrets.SBERCLOUD_CR_PASSWORD }}
+      - name: Build repo image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: condo.tests.Dockerfile
+          push: false
+          tags: ${{ env.DOCKER_IMAGE_FULL }}
+          provenance: false
+          cache-from: type=registry,ref=swr.ru-moscow-1.hc.sbercloud.ru/condo/condo-tests-image:cachekey
+          cache-to: type=registry,ref=swr.ru-moscow-1.hc.sbercloud.ru/condo/condo-tests-image:cachekey,mode=max,image-manifest=true
```

Comment:
IMO it would be better to move this strings to gh secret values

Reply:
IDK, took it from tests job